app/scripts/views/marketing_snippet.js

```diff
@@ -44,7 +43,7 @@ define([
       var marketingType = this.$('[data-marketing-type]').attr('data-marketing-type');
       var marketingLink = this.$('.marketing-link').attr('href');
 
-
+      // TODO: if we have more than one link on the page, what should be logged here?
```

Comment:
Can you remove this, I'll take care of the logging outside of this PR - see #2338